README.md

```diff
@@ -80,68 +36,35 @@ To build and deploy to GitHub Pages:
-
-To deploy your updates to Github Pages, run:
-
-- For Bash:
-
-```console
-   GIT_USER=<GITHUB_USERNAME> yarn deploy
-```
-
-- For Windows
-
-```console
-GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
-```
-
-- For Powershell
-
-```console
-cmd /C 'set "GIT_USER=<GITHUB_USERNAME>" && yarn deploy'
-```
-
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
-
-## Miscellanous
-
-### Tutorial for Converting Table-like Diagrams into Images
-
-To prevent table diagrams overflowing (and having the horizontal scrollbar on narrower viewports),
-we use a simple technique to convert them to images. This allows us to have responsive diagrams
-with less effort and minimizes HTML syntax on our pages.
-
-Tool Recommendation: Firefox Browser (easier to make screenshots)
-
-1. Run the app (`npm run start`)
-
-2. Go to the browser
-
-3. Go where the table you want to convert is
-
-4. Right click, select "Take Screenshot"
-
-![](static/img/take-screenshot.png)
-
-5. Select an area that encloses the whole table
-
-![](static/img/click-to-select-region.png)
+3. Go to the Actions Tab
+   - You will likely see a red x beside a run labelled Initial commit
+   - Click the Initial Commit link
+   - Hit the rerun jobs button
 
-6. Make any adjustments necessary when cropping (avoid having unnecessary space)
+## Files that Require changes:
 
-7. Click Download
+- CNAME
+  - CNAME needs to be changed to match the deployment url of your books. If your book is part of the ICTOER org, then change:
+    `seneca-ictoer.github.io/oertemplate/` to `seneca-ictoer.github.io/yourRepoName/`
+- package.json
+  - Change the name field to match your repo name
+- docusaurus.config.js
+  - Multiple fields that refer to this template needs to change to refer to your repository
 
-![](static/img/download-selected-region.png)
+## Files that you should review
 
-8. Save image to the `/static/img` folder in the repo
+- LICENSE.md - Current license is a generic license. Please update to suit your needs
+- CONTRIBUTE.md - Rewrite to match your contribution workflow
+- README.md - These instructions should modify to suit your OER's intentions
 
-9. This is how it should end up looking like:
+## Prettier
 
-![](static/img/result-of-converted-table.png)
+Remember to run prettier after you make the changes to your files
 
-10. Use markdown to place the image wherever you need:
+## Writing your OER
 
-```
-!["Alt test for accessibility"](/img/name-of-the-image.png)
-```
+- To add content to your OER, edit the files in the docs folder.
+- You can add folders and more md files if you wish
+- Save, add, commit and push your updated files back into github
+- Doing this will automatically start the deployment script which will build the Docusaurus based OER. The URL should be that found in CNAME (although if you didn't change it there will be an error, the url can be found in the repos setting tab->pages)
```

Comment:
Change:  The URL should be that found in CNAME ...
To ->  **The published URL should be what is stated in the CNAME file ...**

Reply:
fixed